README.md

```diff
@@ -57,3 +57,41 @@ jobs:
       shell: bash
 ```
 
+You can disable Nix installation and configure it before running direnv. For example,
+if you use Cachix, you can do:
+
+
+```yaml
+name: use nix
+
+on: push
+env:
+    DEV_SHELL_NAME: CI
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    # Docs to use Cachix: https://nix.dev/tutorials/continuous-integration-github-actions
+    - uses: cachix/install-nix-action@v17
+      with:
+        nix_path: nixpkgs=channel:nixos-unstable
+    - uses: cachix/cachix-action@v11
+      with:
+        name: mycache
+        authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
```

Comment:
Would you want this action to have this step as a configurable option?

Reply:
Well it's up to you. I didn't want to make the scope grow. One might use https://nixbuild.net instead for example, and this example should give enough clues on how to make it compatible.